core/domain/story_services_test.py

```diff
@@ -424,10 +424,16 @@ class StoryProgressUnitTests(StoryServicesUnitTests):
     """
 
     def _get_progress_model(self, user_id, STORY_ID):
+        """Returns the StoryProgressModel corresponding to the story id and user
+        id.
+        """
         return user_models.StoryProgressModel.get(
             user_id, STORY_ID, strict=False)
 
     def _record_completion(self, user_id, STORY_ID, node_id):
+        """Records the completed node in story context corresponding to the
```

Comment:
```suggestion
        """Records the completion of a node in the context of a story. 
```
The "corresponding...." part can be dropped I think.

Reply:
Done